src/tries.jl

```diff
@@ -82,44 +82,6 @@ end
 
 lower(c::UInt8) = UInt8('A') <= c <= UInt8('Z') ? c | 0x20 : c 
 
-"""
```

Comment:
Why move the function? That makes it very hard to spot what this PR changes. If it really needs to be moved, it would be nice to do that in a first commit which doesn't change anything.

Reply:
I haven't actually changed the content of the function I just moved `match!` below the definition of other functions `generatebranch and generatebranches`. Same goes for the second `match!`. Just moved it after definition of other functions. 

The functions are moved because when running with `julia --compiled-modules=no` there is a world age error as `generatebranches` function is defined after `match!` definition. Check out https://github.com/JuliaData/CSV.jl/issues/309 for details

Reply:
OK. Then the commit message should state what it does.

Reply:
commit message changed